### PR TITLE
Python sdk improvements

### PR DESCRIPTION
**Pull request by kp-cat**
### Related tickets

https://trello.com/c/FuPidQQt/160-sdk-loglevel-update-python
https://trello.com/c/i4HzNG0Z/105-sdk-getallvalues-python
https://trello.com/c/v1RVkza1/147-sdk-improvements-python

### Description

Pyhton docs update: logging + timeout + get_all_values + flag_overrides

### Requirement checklist

- [x] I have validated my changes on a test/local environment.
- [ ] I have checked the SNYK/Dependabot reports and applied the suggested changes.
- [ ] (Optional) I have updated outdated packages.

### CODE REVIEW

File: website/docs/sdk-reference/python.md [hunk 1]
```diff
@@ -108,24 +108,27 @@ The *ConfigCat SDK* downloads the latest values and stores them automatically ev
...
 | ----------------------------------- | ---------------------------------------------------------------------------------------------------- | ------- |
 | `poll_interval_seconds`             | Polling interval.                                                                                    | 60      |
-| `on_configuration_changed_callback` | Callback to get notified about changes.                                                              | -       |
 | `max_init_wait_time_seconds`        | Maximum waiting time between the client initialization and the first config acquisition in secconds. | 5       |
+| `on_configuration_changed_callback` | Callback to get notified about changes.                                                              | -       |
 | `config_cache_class`                | Custom cache implementation.                                                                         | None    |
+| `connect_timeout`                   | How many seconds to wait for the server to make the initial connection (i.e. completing the TCP connection handshake). | 10 |
```
Comment (sigewuzhere) [inline website/docs/sdk-reference/python.md:129]:
```suggestion
| `connect_timeout`                   | The number of seconds to wait for the server to make the initial connection (i.e. completing the TCP connection handshake). | 10 |
```

File: website/docs/sdk-reference/python.md [hunk 2]
```diff
@@ -108,24 +108,27 @@ The *ConfigCat SDK* downloads the latest values and stores them automatically ev
...
+| `read_timeout`                      | How many seconds to wait for the server to send data before giving up.                               | 30      |
```
Comment (sigewuzhere) [inline website/docs/sdk-reference/python.md:130]:
```suggestion
| `read_timeout`                      | The number of seconds to wait for the server to respond before giving up.                               | 30      |
```

File: website/docs/sdk-reference/python.md [hunk 3]
```diff
@@ -162,29 +165,65 @@ Available options:
...
+configcat_client = configcatclient.create_client_with_manual_poll('#YOUR-SDK-KEY#')
+configcat_client.force_refresh()
 ```
 
 Available options:
 
 | Option Parameter     | Description                  | Default |
 | -------------------- | ---------------------------- | ------- |
 | `config_cache_class` | Custom cache implementation. | None    |
+| `connect_timeout`    | How many seconds to wait for the server to make the initial connection (i.e. completing the TCP connection handshake). | 10 |
```
Comment (sigewuzhere) [inline website/docs/sdk-reference/python.md:186]: same as above

File: website/docs/sdk-reference/python.md [hunk 4]
```diff
@@ -162,29 +165,65 @@ Available options:
...
+| `read_timeout`       | How many seconds to wait for the server to send data before giving up. | 30 |
```
Comment (sigewuzhere) [inline website/docs/sdk-reference/python.md:187]: same as above

File: website/docs/sdk-reference/python.md [hunk 5]
```diff
@@ -162,29 +165,65 @@ Available options:
...
+| `flagOverrides`      | Configures local feature flag & setting overrides. [More about feature flag overrides](#flag-overrides) | None |
```
Comment (sigewuzhere) [inline website/docs/sdk-reference/python.md:188]: Try to avoid the word "configure"

**Review by sigewuzhere — CHANGES_REQUESTED** [review]

**Review by sigewuzhere — APPROVED** [review]

**Review by z4kn4fein — APPROVED** [review]

**Pull request merged**